R/cded.R

```diff
@@ -49,6 +50,8 @@ cded <- function(aoi = NULL, mapsheets = NULL, .predicate = sf::st_intersects, d
       stop("You have entered invalid mapsheets", call. = FALSE)
     }
   } else {
+    if (sf::st_crs(aoi)$input != "EPSG:3005") aoi <- st_transform(aoi, sf::st_crs(mapsheets_250K()))
```

Comment:
I think you could just use `transform_bc_albers()`... It basically just calls `st_transform()`, which only does the transformation if the CRSs are different otherwise it just returns the input.